os/adding-swap.md

```diff
@@ -2,63 +2,61 @@
 
 Swap is the process of moving pages of memory to a designated part of the hard disk, freeing up space when needed. Swap can be used to alleviate problems with low-memory environments.
 
-By default Container Linux does not include a partition for swap, however one can configure their system to have swap.
+By default Container Linux does not include a partition for swap, however one can configure their system to have swap, either by including a dedicated partition for it or creating a swapfile.
 
 ## Managing swap with systemd
 
-A systemd-managed service can be used to create, enable, and disable a swapfile service on your system. Write, enable, and start the systemd unit file found in the next section to add swap to a Container Linux node.
+systemd provides a specialized `.swap` unit file type which may be used to activate swap. The below example shows how to add a swapfile and activate it using systemd.
 
-### Creating the systemd unit file
+### Creating a swapfile
 
-The following systemd unit creates and enables a service to manage a 1GiB swapfile. It should be written to `/etc/systemd/system/swap.service`.
+The following commands will make a 1GiB file suitable for use as swap.
 
+```sh
+mkdir -p /var/vm
```

Comment:
All of these commands implicitly assume root.  Do we want to say that?

Reply:
Added `, as root, ` to the line above. For the others (using systemd, writing a unit) I think we frequently assume root in other docs and it's a fairly clear error / systemd login prompt / whatever if you omit it.